elkscmd/gui/render.c

```diff
@@ -11,7 +11,7 @@
 // ----------------------------------------------------
 void R_ClearCanvas(void)
 {
-    fillrect(0, 0, SCREEN_WIDTH-1, SCREEN_HEIGHT-1, BLACK);
+    fillrect(0, 0, SCREEN_WIDTH, SCREEN_HEIGHT-1, BLACK);
```

Comment:
I still don't quite understand this change, this will clear from 0 up to and including SCREEN_WIDTH wide. Isn't that 1 too many? The canvas mouse X clipping above is to SCREEN_WIDTH-1, for instance. This isn't clearing the white dividing line though, so I guess it is working. I tried the previous version and it didn't seem to have an issue though.

Reply:
Apparently we draw vertical white line at screen_width + 1. 
Btw, why don’t we name it canvas_width as opposed to screenwidth?

Reply:
> why don’t we name it canvas_width as opposed to screenwidth?

That's a great idea, especially considering that SCREEN_WIDTH is so close to SCREENWIDTH (the actual graphical screen width), which were globals from the original graphics.c I pulled over from 8086-toolchain/examples. Go ahead and make that change to with your further enhancements, thank you.

> Apparently we draw vertical white line at screen_width + 1.

I see. That probably means we can't actually draw a single pixel brush at SCREEN_WIDTH, since it's clipped.